src/test/java/core/basesyntax/dataprocessor/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.dataprocessor;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FruitDB;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PurchaseHandlerTest {
```

Comment:
Let's use the same fruit for basic tests and move it to the constants

Reply:
fix comment